en/docs/insights/view-api-insights.md

```diff
@@ -22,7 +22,7 @@ The information displayed is as follows:
 
   ![Total traffic](../assets/img/insights/overview-page-total-traffic.png){.cInlineImage-half}
 
-  This widget displays the total traffic your selected environment received during the selected time range. This includes both successful requests and error requests. To investigate further, you can click the arrow icon on the bottom right corner of the widget to open the [**Traffic** page](#traffic)
+  This widget displays the total traffic your selected environment received during the selected time range. This includes both successful requests and error requests. To investigate further, you can click the arrow icon on the bottom right corner of the widget to open the [**Traffic Page**](#traffic).
```

Comment:
Why was **Traffic** page changed to **Traffic Page**?

Reply:
Isn't it bit odd to make page  'p' in simple and Traffic 'T' in capital? And I guess we need to bold Traffic Page as whole and not only Traffic.